recipes/pandas-alive/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pandas-alive" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pandas_alive/pandas_alive-{{ version }}.tar.gz
+  sha256: d88a8d81baafda50b77dfdc5bdf62b9c9d2f2d4ae53a031b168c244d7d80907b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```

Comment:
Sorry for what looks like some confusion about my original comment. I was saying that the `build` section under `requirements` should be removed, not the top-level one.

Reply:
All good, I got there haha 😄